subsystems/drivebase/SwerveDrive.java

```diff
@@ -59,6 +61,11 @@ public abstract class SwerveDrive extends DriveBase {
     private final Mk4ModuleConfiguration kModuleConfigBackRight = new Mk4ModuleConfiguration();
     private final double kPitchOffset;
     private final double kRollOffset;
+    // private final double currentAlpha; // TODO tune, closer to 1 gives recent readings more weight, closer to 0 gives older readings more weight
```

Comment:
if you're switching to a different approach then delete the old one, don't just leave it here commented out